app/assets/javascripts/components/CartedProducts.jsx

```diff
@@ -79,7 +91,9 @@ var CartedProducts = React.createClass({
             )}
           </tbody>
         </table>
+        <hr/>
         <div>
+          <p><b>Total: </b>{(this.state.cart_total * 0.01).toLocaleString("en-US", {style: "currency", currency: "USD", minimumFractionDigits: 2})}</p>
```

Comment:
Maybe there's a function we could have that does this: `* 0.01).toLocaleString("en-US", {style: "currency", currency: "USD", minimumFractionDigits: 2})`, since it's a conversion we'll need on almost every page? Just something to think about in a future pull request.